adapter-http/src/main/resources/activities/baselines/http-rest-timeseries.yaml

```diff
@@ -27,10 +26,10 @@ bindings:
 
   # http request id
   request_id: ToHashedUUID(); ToString();
-  request_token: ToString(); TextOfFile("TEMPLATE(stargate_tokenfile,data/stargate_token.txt)")
+  request_token: ToString(); TextOfFile(TEMPLATE(stargate_tokenfile,'data/stargate_token.txt'));
```

Comment:
quote placement (move outside of TEMPLATE)

Reply:
Added.